tests/test_browser.py

```diff
@@ -3053,6 +3053,9 @@ def test_canvas_style_proxy(self):
   def test_canvas_size_proxy(self):
     self.btest(path_from_root('tests', 'canvas_size_proxy.c'), expected='0', args=['--proxy-to-worker'])
 
+  def test_custom_messages_proxy(self):
+    self.btest(path_from_root('tests', 'custom_messages_proxy.c'), expected='1', args=['--proxy-to-worker', '--shell-file', path_from_root('tests/custom_messages_proxy_shell.html'), '--post-js', path_from_root('tests/custom_messages_proxy_postjs.js')])
```

Comment:
`path_from_root('tests/custom_messages_proxy_shell.html')` should be `path_from_root('tests', 'custom_messages_proxy_shell.html')` so that it works on windows